dev/ToastNotifications/ToastNotificationManager.cpp

```diff
@@ -102,9 +107,21 @@ namespace winrt::Microsoft::Windows::ToastNotifications::implementation
     {
         GetToastHandlers().remove(token);
     }
-    void ToastNotificationManager::ShowToast(winrt::Microsoft::Windows::ToastNotifications::ToastNotification const& /* toast */)
+    void ToastNotificationManager::ShowToast(winrt::Microsoft::Windows::ToastNotifications::ToastNotification const& toast)
     {
-        throw hresult_not_implemented();
+        THROW_HR_IF(WPN_E_NOTIFICATION_POSTED, toast.ToastId() != 0);
```

Comment:
This is great. It prevents misuse from the developer by preventing them from displaying the same toast again and again!